### PR TITLE
Truncate random seed to fit into int during protobuf serialization

### PR DESCRIPTION
**Pull request by maximsch2**
Fixes https://github.com/tensorflow/tensorflow/issues/2460

### CODE REVIEW

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

File: tensorflow/python/framework/random_seed.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 
 _DEFAULT_GRAPH_SEED = 87654321
 
+def _truncate_seed(seed):
+  return seed % 2147483647 # truncate to fit into 32-bit integer
```
Comment (zheng-xq) [inline tensorflow/python/framework/random_seed.py]: Please use 2 ** 31 -1 instead.

Comment (maximsch2) [inline tensorflow/python/framework/random_seed.py]: Replaced it with a constant. Couldn't find a pre-defined constant for this in Python....

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

**Comment by vrv** [comment]
Looks like flakiness in matmul test, trying again.  @tensorflow-jenkins test this please

**Pull request merged**